PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -158,25 +161,29 @@ def _tauId8WPMask(pattern,doc):
 
 tauTable.variables = _variablesMiniV2
 
+(run2_nanoAOD_92X | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_94X2016 | run2_nanoAOD_102Xv1 | run2_nanoAOD_106Xv1).toModify(tauTable,
+                 variables = cms.PSet(tauTable.variables, _mvaAntiEVars, _mvaIsoVars2015Reduced, _mvaIsoVars2017v1, _mvaIsoVars2017v2)
```

Comment:
we removed the _mvaAntiEVars in the master, why not here as well ?

Reply:
Just for completeness: Concerning the line above in particular, this required to restore mvaAntiEVars since it is not done by a following modifier.